livekit-agents/livekit/agents/ipc/proc_pool.py

```diff
@@ -20,7 +21,7 @@
     "process_job_launched",
 ]
 
-MAX_CONCURRENT_INITIALIZATIONS = 5
+MAX_CONCURRENT_INITIALIZATIONS = os.cpu_count() or 1
```

Comment:
this isn't container aware, we need to use our capacity util here: https://github.com/livekit/agents/blob/main/livekit-agents/livekit/agents/utils/hw/cpu.py